action_list: 
    number_of_targets: 3

    action_0: 
        action_type: "pose"
        translation: [0.40, 0.15, 0.4]
        orientation: [0.0, 0.0, 0.0]
        duration: 3.0

    action_1: 
        action_type: "pose"
        translation: [0.40, 0.15, 0.35]
        orientation: [0.0, 0.0, 0.0]
        duration: 1.0

    action_2:
        action_type: "pose"
        translation: [0.40, -0.20, 0.35]
        orientation: [0.0, 0.0, 0.0]
        duration: 2.0